src/snowflake/snowpark/modin/plugin/_internal/align_utils.py

```diff
@@ -93,3 +97,120 @@ def align_axis_0_right(
             right_frame_data_ids + right_index_ids
         )
     return right_result, right_frame, right_frame_data_ids, right_index_ids
+
+
+def align_axis_1(
+    frame1: InternalFrame,
+    frame2: InternalFrame,
+    join: Literal["inner", "outer", "left", "right"],
+    sort: Optional[bool] = False,
+) -> tuple[InternalFrame, InternalFrame]:
+    """
+    Concatenate frames on index axis by taking using UNION operator.
```

Comment:
```suggestion
    Concatenate frames on index axis using UNION operator.
```

Reply:
The comments/ functions have been changed quite a bit since!